sync/cli/_databricks.py

```diff
@@ -188,11 +188,13 @@ def get_cluster_report(
 @click.argument("job-id")
 @click.argument("project-id")
 @click.argument("recommendation-id")
+@click.argument("workspace-id")
 @pass_platform
 def apply_recommendation(
     platform: Platform,
     job_id: str,
     project_id: str,
+    workspace_id: str,
```

Comment:
Do we need to let cli users know this is required now?

Reply:
+1